linux_os/guide/services/xwindows/xwindows_base_packages.var

```diff
@@ -0,0 +1,24 @@
+documentation_complete: true
```

Comment:
I think this file is some remains of a different approach you were trying

Reply:
Dropped in  [24a75b4](https://github.com/ComplianceAsCode/content/pull/12853/commits/24a75b40f72baaa41a8c1b064618343c771efc9b) :bow: